server/src/queues/setEnvVar.ts

```diff
@@ -0,0 +1,48 @@
+import { Worker, Queue } from 'bullmq';
```

Comment:
@pradel my first attempt at creating queue! It works as I understood it should :  mutation now takes almost no time and env vars are set correctly when I check them. And currently we don't produce any logs or other feedback, but as I understood that was the plan for this one. 

If this is good, I will create one for `unsetEnvVar` as well.

Reply:
@Akirtovskis Yup you are right, I guess the best will be to notify the user that the app will be redeployed when the env is changing and add a new entry in the activity list :)

Reply:
Looks good!

Reply:
@pradel unsetEnvVar queue added, all the feedbacks are fixed and naming is improved to map to 🐳 . Should be good to go, but I am happy to fix if something is still no bueno.